signing-manifests/2023-new-gpg-subkey-test.yml

```diff
@@ -0,0 +1,12 @@
+---
+bug: 6666666666
+sha256: 570d80410aa9333a3749bd538041a0456bf78e83ad27b485aa89f19c07d03890
+filesize: 79666542
+private-artifact: false
+signing-formats: [ "autograph_gpg" ]
+requestor: bhearsum
+reason: test new gpg signing key
+artifact-name: signed.zip
```

Comment:
I believe this should be `firefox-113.0.tar.bz2`, IIRC we don't support repackaging tarballs, only zips.